stacks/php-laravel/devfile.yaml

```diff
@@ -27,7 +28,7 @@ components:
       endpoints:
         - name: http-laravel
           targetPort: 8000
-      image: quay.io/devfile/composer:2.1.11
+      image: quay.io/devfile/composer:latest
```

Comment:
Is it possible to use something other than latest. Something like `composer:2` or `composer:2.4` The reason we changed the composer image away from latest was that it was bumped to a PHP version that broke the starter projects in the devfile, and I'd like to avoid that issue happening again the next time PHP is updated.

Reply:
Also, stack version should be bumped

Reply:
Also, this image has [several critical vulnerabilities](https://quay.io/repository/devfile/composer/manifest/sha256:110fff023c715eb30fc8a2f4806b5b966cfff5c6104d1396ab73c78a3524083d?tab=vulnerabilities) :-(

Reply:
> We could probably limit it to composer:2; however, we don't do this for any other stacks. We also have tests that would detect if it failed.

Yes, but ideally we can avoid getting broken in the first place.

Reply:
>Sadly there isn't anything we can do about this because RH doesn't have a composer image.

Who maintains this image?